docs/getting-started/bitfinity-engine.md

```diff
@@ -4,19 +4,17 @@ title: "Bitfinity Engine"
 
 # Introduction
 
-[Bitfinity](https://bitfinity.network) is an Ethereum Virtual Machine (EVM) built on the
+[The Bitfinity Network EVM](https://bitfinity.network) is an Ethereum Virtual Machine (EVM) built on the
 [Internet Computer Protocol](https://internetcomputer.org/),
 that provides a solution for developers to deploy their apps on an Ethereum-compatible, high-throughput, scalable and future-safe platform,
-with low transaction costs for their users. Besides the EVM, Bitfinity developed the Bitfinity Bridge which allows users to transfer assets between Ethereum, Internet Computer, and Bitfinity.
-Bitfinity is backed by top VCs such as Polychain Capital Beacon Fund, ParaFi Capital, Draft Ventures.
+with low transaction costs for their users. In addition to the Bitfinity Network EVM, Bitfinity has developed the Bitfinity Bridge which allows users to transfer assets between the Ethereum network, the Internet Computer network, and the Bitfinity network.
+This bridge is built using canister smart contract technology exclusive to the Internet Computer, and the custody of assets is fully decentralised thanks to Chain-Key technology.
+Bitfinity is backed by top VCs such as Polychain Capital Beacon Fund, ParaFi Capital, and Draft Ventures.
 
-Thanks to Bitfinity’s EVM, Ethereum native applications can seamlessly be ported to Internet Computer’s L2-like network that is built as a smart contract on Internet Computer.
-Developers may enjoy familiar Ethereum tooling when working with their Solidity smart contracts on Bitfinity.
-The base fee of Bitfinity is ETH, which provides a smooth experience for dapps’ users.
```

Comment:
Having read the FAQ it looks as though the base fee should indeed be listed as Bitfinity Token (BFT), and that change should be reflected here on line 15